monitors/SyntheticPegMonitor.js

```diff
@@ -0,0 +1,100 @@
+const { createFormatFunction, createEtherscanLinkMarkdown } = require("../common/FormattingUtils");
+
+class SyntheticPegMonitor {
+  /**
+   * @notice Constructs new Liquidator bot.
+   * @param {Object} logger Module used to send logs.
+   * @param {Object} uniswapPriceFeed Module used to query the current uniswap token price.
+   * @param {Object} cryptoWatchPriceFeed Module used to query the current crypto watch token price.
+   * @param {Object} [config] Contains fields with which constructor will attempt to override defaults.
+   */
+  constructor(logger, uniswapPriceFeed, cryptoWatchPriceFeed) {
+    this.logger = logger;
+
+    // Instance of price feeds used to check for deviation of synthetic token price.
+    this.uniswapPriceFeed = uniswapPriceFeed;
+    this.cryptoWatchPriceFeed = cryptoWatchPriceFeed;
+
+    this.web3 = this.uniswapPriceFeed.web3;
+
+    // Contract constants
+    // TODO: replace this with an actual query to the collateral currency symbol
+    this.collateralCurrencySymbol = "DAI";
+    this.syntheticCurrencySymbol = "UMATEST";
+
+    // TODO: get the decimals of the collateral currency and use this to scale the output appropriately for non 1e18 colat
+    this.formatDecimalString = createFormatFunction(this.web3, 2);
+
+    // If the price of one price feed deviates by more than `deviationAlertThreshold` percent from the other, trigger alert.
+    this.deviationAlertThreshold = this.web3.utils.toBN(this.web3.utils.toWei("20"));
```

Comment:
Once #1430 is merged you should define default values for `deviationAlertThreshold`, `collateralCurrencySymbol`, `syntheticCurrencySymbol`, `decimals`, etc.

Reply:
Ah I see you do this. You should follow the pattern in `liquidator/Liquidator.js` where you define the `defaultConfig` before you set the config properties. In this case, you could set `value = toBN(toWei("20"))` and `isValid = () => {return x.gte(toBN(0)) && x.lt(toBN(100))}`.

Remember to call `Object.assign(this, createObjectFromDefaultProps)` at the end to set the config properties to `this`